tests/active_features/connection:info.feature

```diff
@@ -0,0 +1,64 @@
+@environment @connection
+Feature: Environment Connection Info Command
+  In order to see connection parameters for remotely accessing the environment of a given site
+  As a Terminus user
+  I want a command to view the current connection parameters.
+
+  Background: I am authenticated and have a site named [[test_site_name]]
+
+  @vcr site_connection-info
+  Scenario: Show all connection info for a site's environment
+    Given I am authenticated
+    And a site named: [[test_site_name]]
+    When I run "terminus connection:info [[test_site_name]].[[test_env_name]]"
+    Then I should see a table with the headers: Parameter, Connection Info
+    And I should see a table with rows like:
```

Comment:
"with rows like" matches as long as each line in the table contains the corresponding expected string somewhere on the line?